src/Controllers/Library/search.php

```diff
@@ -1,6 +1,54 @@
 <?php
-
+/**
+ * Allows URY Librarians to search for Tracks.
+ */
 use \MyRadio\MyRadio\CoreUtils;
+use \MyRadio\MyRadio\URLUtils;
+use \MyRadio\ServiceAPI\MyRadio_Track;
+use \MyRadio\MyRadio\MyRadioForm;
+use \MyRadio\MyRadio\MyRadioFormField;
+
+$form = new MyRadioForm(
+    'lib_search',
+    'Library',
+    'search',
+    [
+        'title' => 'Library Search',
+    ]
+);
+
+$form->addField(
+    new MyRadioFormField('title', MyRadioFormField::TYPE_TEXT, ['required' => false, 'label' => 'Title', 'placeholder' => 'Filter by track title...'])
+)->addField(
+    new MyRadioFormField('artist', MyRadioFormField::TYPE_TEXT, ['required' => false, 'label' => 'Artist', 'placeholder' => 'Filter by artist name...'])
+);
+        
+if ($_SERVER['REQUEST_METHOD'] === 'POST') {
+    //Submitted
+    $data = $form->readValues();
+
+    if (isset($data['title']) || isset($data['title'])) {
+        $tracks = MyRadio_Track::findByOptions(
+            [
+                    'title' => isset($data['title']) ? $data['title'] : '',
+                    'artist' => isset($data['artist']) ? $data['artist'] : '',
+                    'limit' => 30
```

Comment:
It does make sense, but then if you are searching the library you probably want to search the whole library. I don't see much reason to limit it, other than you could get a *lot* of data back with a silly search. The standard way to do that would be to only fetch a page of results at a time, but we don't easily support that in MyRadio.

I think seeing as this is the place to search the library it might as well not be limited. Others may dispute this though.

Maybe leave it for now, it's easy enough to remove later.